### ISSUE TITLE
Error when compiling a namespace with `:require-macros` and `:refer`

### ISSUE THREAD

**Issue by Akeboshiwind** [squint-cljs/squint#449]
**version**

`0.4.84`

**problem**

I'm getting `Error: fileURLToPath is not ISeqable` when trying to `(:require [xxx :refer [xxx]])` and `:require-macros` in the same ns.

**repro**

Project setup:

```bash
$ mkdir squint-test && cd squint-test
$ npm init -y
$ npm install squint-cljs@latest
```

<details><summary>squint.edn</summary>

```clojure
{:paths ["src"]}
```

</details>

<details><summary>src/macro_scratch.cljc</summary>

```clojure
(ns macro-scratch)

(defmacro my-print [value]
  `(println ~value))
```

</details>

<details><summary>src/example.cljs</summary>

```clojure
(ns example
  (:require-macros [macro-scratch :refer [my-print]])
  (:require ["url" :refer [fileURLToPath]]))

(println (fileURLToPath js/import.meta.url))
```

</details>

Repro:
```bash
$ npx squint compile
```

<details><summary>Full output</summary>

```bash
[squint] Compiling CLJS file: /Users/osm/prog/prog/clojure/squint/src/example.cljs
file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:42
$APP.x=function(a){if(null==a)return null;if(null!=a&&(a.D&8388608||$APP.Oc===a.Ee))return a.Y(null);if($APP.Xa(a)||"string"===typeof a)return 0===a.length?null:new $APP.Xc(a,0,null);if(null!=a&&null!=a[Zc])return a=$APP.xa(a,Zc).call(a),ad.h?ad.h(a):ad.call(null,a);if($APP.cb(bc,a))return $APP.cc(a);throw Error([$APP.t.h(a)," is not ISeqable"].join(""));};$APP.y=function(a){if(null==a)return null;if(null!=a&&(a.D&64||$APP.Oc===a.ad))return a.Fa(null);a=$APP.x(a);return null==a?null:rb(a)};
                                                                                                                                                                                                                                                                                                                    ^

Error: fileURLToPath is not ISeqable
    at $APP.x (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:42:309)
    at $APP.g.cd (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:658:254)
    at nc (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:30:45)
    at $APP.g.Ia (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:605:243)
    at $APP.oe.j (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:440:74)
    at $APP.Rg.g (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:582:97)
    at Function.d [as g] (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:519:159)
    at $APP.Jk.A (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:591:456)
    at $APP.Jk.A (file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:591:385)
    at file:///Users/osm/prog/prog/clojure/squint/node_modules/squint-cljs/lib/compiler.js:268:271

Node.js v21.1.0
```

</details>

**expected behavior**

To be able to import both a macro and a `:refer`d function

**Comment by borkdude** [comment]
Hmm, I do have test project for this, but I'll look into this!

**Comment by borkdude** [comment]
Yep, can reproduce!

**Comment by borkdude** [comment]
Fixed in 0.4.85
